### ISSUE TITLE
Request of credentials

### ISSUE THREAD

**Issue by piranna** [stefanzweifel/git-auto-commit-action#67]
**Version of the Action**
`v4.1.5`

**Describe the bug**
After upgrading to latest version of the Action, I'm getting an error about needing to define my identity:

```
[master 9f54f5b] 2020.4.26
 Author: piranna <piranna@users.noreply.github.com>
 4 files changed, 22 insertions(+), 6 deletions(-)
INPUT_TAGGING_MESSAGE: 2020.4.26

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az16.rubf4iz3or0uhhn53zl4cow15h.bx.internal.cloudapp.net>) not allowed
Error: Invalid status code: 128
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4.1.5/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
  code: 128
}
Error: Invalid status code: 128
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4.1.5/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

You can find my workflow at https://github.com/projectlint/OS-lifecycle/blob/master/.github/workflows/nightly.yml and full Action execution at https://github.com/projectlint/OS-lifecycle/runs/625667062

**Comment by piranna** [comment]
Problem is happening when doing the `push`, other commands works are expected.

**Comment by stefanzweifel** [comment]
Thanks for reporting.
Seems the change made in #59 is related to this.

Will fix this shortly.

**Comment by piranna** [comment]
> Seems the change made in #59 is related to this.

Yes, it seems so, probably it's just a matter of add the same parameters.

**Comment by stefanzweifel** [comment]
Fixed in v4.1.6: https://github.com/stefanzweifel/git-auto-commit-action/releases/tag/v4.1.6

**Comment by piranna** [comment]
I thought it was about the push, but yes, it was the tags, thank you :-)
